### PR TITLE
password toggle feature

### LINKED ISSUE #18: Feat: Toggle View for Password field during signup and login
### Describe the feature

The field currently shows as password field which can be modified to have a show/hide toggle 

### Add ScreenShots

image

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I'm a GSSOC'24 contributor
- [ ] I want to work on this issue

### PR DESCRIPTION
**Pull request by Arya-Hari**
## PR Description

_Add your description_

Adding a password toggle feature for the login page

## Related Issues: Issue for which you are raising a PR for

- Closes #18 

### Issue

[Issue number or description]

### Checklist

- [x] I have gone through the [contributing guide](https://github.com/SaiyamTuteja/Faculty_Evaluation_System/blob/main/CONTRIBUTING.md)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [ ] Is this a bug fix/enhancement/documentation changes
- [x] Part of GSSOC
- [ ] Tested for any breaking changes
- [ ] Other relevant checks completed

## Undertaking

I declare that: 

1. The content I am submitting is original and has not been plagiarized.  
2. No portion of the work has been copied from any other source without proper attribution.  
3. The work has been checked for plagiarism, and I assure its authenticity.  

I understand that any violation of this undertaking may have legal consequences that I will bear and could result in the withdrawal of any recognition associated with the work. 

- [x] I Agree

### CODE REVIEW

File: login.php [hunk 1]
```diff
@@ -63,7 +63,7 @@
           <input type="password" class="form-control" name="password" required placeholder="Password">
           <div class="input-group-append">
             <div class="input-group-text">
-              <span class="fas fa-lock"></span>
+              <span class="fas fa-eye"></span>
```
Comment (aswinikalyan30) [inline login.php:66]: I dont see the `password-toggle` added here - Where will the function come? Is there any onClick handler you wanted to add?

**Review by aswinikalyan30 — CHANGES_REQUESTED** [review]

File: login.php (hunk 1 above)
Comment (Arya-Hari) [reply login.php:66]: Hi sorry I've made an update to add an id. Please check it out.

File: login.php [hunk 2]
```diff
@@ -54,7 +54,7 @@
         <div class="input-group mb-3">
           <input type="email" class="form-control" name="email" required placeholder="Email">
           <div class="input-group-append">
-            <div class="input-group-text">
+            <div class="input-group-text" id="password-toggle">
```
Comment (aswinikalyan30) [inline login.php:57]: This won't work - you are adding the id to the wrong icon and It is not being made a clickable attribute 
After making changes - please add screenshots to the PR

File: login.php [hunk 3]
```diff
@@ -127,6 +127,18 @@
     })
   })
   })
+
+  $(document).ready(function(){
```
Comment (aswinikalyan30) [inline login.php:131]: Move the function inside the already existing document.ready - don't need it twice

**Review by aswinikalyan30 — CHANGES_REQUESTED** [review]

**Comment by aswinikalyan30** [comment]
Hi @Arya-Hari - any progress?

**Comment by aswinikalyan30** [comment]
Closing this as it has been solved by someone else.

**Pull request closed without merging**